File: 0001 Code Chef/0016 Minimum Wage/implementation/0010 nodejs/minimum_wage.js
const readline = require('readline');


const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let minimum_wage = 11;
    let input_wage = parseInt(input);
    if(input_wage > minimum_wage){
            console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
})

File: 0001 Code Chef/0016 Minimum Wage/implementation/0001 c/minimum_wage.c
#include <stdio.h>

int main(){
    int minimum_wage,input_wage;
    minimum_wage = 11;
    scanf("%d",&input_wage);
    if(input_wage > minimum_wage){
        printf("YES\n");
    }else{
        printf("NO\n");
    }
    return 0;
}